client/src/util/clamp.js

```diff
@@ -0,0 +1,3 @@
+export default function clamp(val, rng) {
```

Comment:
Please add a comment that describes the purpose of the function, the parameters and the return value.  For example:
```
/**
  * The function will ...
  * @param x - a d3 object which ....
  * @returns - ...
  */
```